### PR TITLE
Change some binning, kill some useless sparse, overall make code more lightweight

### PR DESCRIPTION
**Pull request by fibnumb**

### CODE REVIEW

File: PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA.cxx [hunk 1]
```diff
@@ -784,17 +802,26 @@ void AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA::DoTrackLoop()
...
+    AliTrackContainer* trackCont = GetTrackContainer("tracks");
+   
     TString histname;
     TString groupname;
     AliParticleContainer* partCont = 0;
     TIter next(&fParticleCollArray);
+    
+    for (auto trackIt : trackCont->accepted_momentum() ) {
+        
+    }
```
Comment (raymondEhlers) [inline PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetSpectra8TeVTriggerQA.cxx]: Did you mean to leave this here?

**Comment by raymondEhlers** [comment]
+test

**Comment by raymondEhlers** [comment]
+1

**Comment by fibnumb** [comment]
Thanks.  I commented it out for now

**Comment by fibnumb** [comment]
Could someone please proceed with a test?

**Comment by raymondEhlers** [comment]
+1. Apparently it parsed my +1 before your commit even though I did it before

**Comment by fibnumb** [comment]
no prob.  Thanks!!

**Pull request merged**